### PR TITLE
Added information about SymPy Gamma and SymPy Live | ashutosh.saboo96@gmail.com

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
Hello everyone. 

I have added some details about SymPy Gamma and SymPy Live. 

Please review the same @asmeurer @certik . Please provide feedback, in case if any extra information needs to be added. It'll be really helpful for me.

Thanks! :smile:

### CODE REVIEW

File: other_projects_that_use_sympy.tex [hunk 1]
```diff
@@ -0,0 +1,38 @@
+\subsection{SymPy Gamma}\label{sympy-gamma}
+
+SymPy Gamma is a simple web application based on Google App Engine that
```
Comment (moorepants) [inline other_projects_that_use_sympy.tex]: based on > that runs on

File: other_projects_that_use_sympy.tex [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+\begin{itemize}
+\item
+  It displays integration steps, differentiation steps in detail, which
+  can be viewed through this
+  \href{http://www.sympygamma.com/input/?i=integrate\%281\%20/\%28\%28x\%2B1\%29\%28x\%2B3\%29\%28x\%2B5\%29\%29\%29}{Link}.
+\item
+  It also displays the factor tree diagrams for numbers, which is also
+  illustrated through this
+  \href{http://www.sympygamma.com/input/?i=112}{link}.
```
Comment (moorepants) [inline other_projects_that_use_sympy.tex]: Screenshots might be better here as they add permanency to the content of the paper.

**Comment by ashutoshsaboo** [comment]
@moorepants Thanks for your feedback. Yes, I agree with you that images must be used instead of links for permanency. 

I have made the required changes. Please suggest feedback on the same. @moorepants @asmeurer @certik 

Does it look good now?

File: other_projects_that_use_sympy.tex [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+entering an arbitrary expression will provide the derivative, integral,
+series expansion, plot, and roots.
+
+SymPy Gamma also has several additional features than just computing the
+results using SymPy.
+
+\begin{itemize}
+\item
+  It displays integration steps, differentiation steps in detail, which
+  can be viewed below:\par
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: Refer to the image using a `\ref`

File: other_projects_that_use_sympy.tex [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  illustrated through this image:\par
+\begin{minipage}{\textwidth}
+    \centering
+    \includegraphics[width=0.7\textwidth]{factor_tree_diagram.png}
+    \captionof{figure}{Factor Tree Diagram of \textbf{112}}
+\end{minipage}
+\item
+  SymPy Gamma also saves user search queries, and offers many such 
+  similar features for free, which Wolfram\textbar{}Alpha only offers 
+  to it's paid users.
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: its

File: other_projects_that_use_sympy.tex [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+\end{itemize}
+Every input query from the user on SymPy Gamma is first, parsed by it's
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: its

File: other_projects_that_use_sympy.tex [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+results using SymPy.
+
+\begin{itemize}
+\item
+  It displays integration steps, differentiation steps in detail, which
+  can be viewed below:\par
+\begin{minipage}{\textwidth}
+    \centering
+    \includegraphics[width=0.7\textwidth]{der_steps-min.png}
+    \captionof{figure}{Derivative steps of $\frac{\mathrm{d}}{\mathrm{d} x}\left (\frac{-log(x + 3)}{4} + \frac{log(x^2 + 6x + 5)}{8}\right)$}
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: Use `\log`

**Comment by asmeurer** [comment]
The images are too big. I would find a smaller derivative or integral steps example, and discard the integer image.

**Comment by ashutoshsaboo** [comment]
Hi @asmeurer. Thanks for your feedback. I'll make the changes and push the new commit. 

Actually for the integral steps I thought that we must put a screenshot of some integral involving too many steps, which would also in a way show that SymPy Gamma can evaluate the huge integral steps as well. Though I understand your point. 

So, should I replace the integral steps image? Also, should I remove the factor tree diagram image? 

Thanks. :smiley:

**Comment by ashutoshsaboo** [comment]
@asmeurer So, I have made the required changes. And it has also passed all the Travis tests. So, should I change the images?

**Comment by ashutoshsaboo** [comment]
@asmeurer @certik @moorepants I have also added some details about SymPy Live. Please review the commit. Please provide feedback on the same. 

Thanks. :smile:

File: other_projects_that_use_sympy.tex [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+\end{minipage}
+\item
+  SymPy Gamma also saves user search queries, and offers many such 
+  similar features for free, which Wolfram\textbar{}Alpha only offers 
+  to its paid users.
+\end{itemize}
+Every input query from the user on SymPy Gamma is first, parsed by its
+own parser, which handles several different forms of function names,
+which SymPy as a library doesn't support. For instance, SymPy Gamma
+supports queries like - \texttt{sin\ x}, whereas SymPy doesn't support
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: Remove the dash

File: other_projects_that_use_sympy.tex [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+this, and supports only \texttt{sin(x)}.
+
+This parser, then converts the input query to the resultant SymPy
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: Remove the comma

File: other_projects_that_use_sympy.tex [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+readable code, which is then eventually processed by SymPy and the
+result is finally formatted in LaTeX and displayed on the SymPy Gamma
+web-app.
+
+\subsection{SymPy Live}\label{sympy-live}
+
+Sympy Live, an online shell for SymPy is a simple web application that
+runs on Google App Engine, which allows to evaluate Python code with
+SymPy in web browsers.
```
Comment (asmeurer) [inline other_projects_that_use_sympy.tex]: This sentence is a fragment.

**Comment by asmeurer** [comment]
For SymPy Live, I would just mention that it runs SymPy in an online Python shell, and that it is integrated in the SymPy documentation examples at docs.sympy.org.

**Comment by asmeurer** [comment]
Regarding the images, yes, I would get rid of the integer one and find a much smaller one for a derivative or integral. We have limited space for the paper (20 pages for the SIAM journal), so large images or figures should only be used if they add significant content to the paper.

**Comment by ashutoshsaboo** [comment]
@asmeurer 

Regarding the images, I do agree with you since, the limit is of 20 pages, that you mentioned. We must only include the necessary images only. 

I have updated the commits with the changes you suggested. Thanks for your feedback. Any other changes that you'd advice me?

Thanks! :smile:

**Comment by ashutoshsaboo** [comment]
So, does this seem fine and final? @asmeurer

File: other_projects_that_use_sympy.tex [hunk 10]
```diff
@@ -0,0 +1,70 @@
+\subsection{SymPy Gamma}\label{sympy-gamma}
+
+SymPy Gamma is a simple web application that runs on Google App Engine 
+that executes and displays the results of SymPy expressions as well as
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: runs on Google App Engine that executes --> runs on Google App Engine. It executes

File: other_projects_that_use_sympy.tex [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+additional related computations, in a fashion similar to that of
+Wolfram\textbar{}Alpha. For instance, entering an integer will display
+prime factors, digits in the base-10 expansion, and a factorization
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: prime factors --> its prime factors

File: other_projects_that_use_sympy.tex [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+diagram. Entering a function will give its docstring; in general,
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: give --> display

File: other_projects_that_use_sympy.tex [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+entering an arbitrary expression will provide the derivative, integral,
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: provide --> display its

File: other_projects_that_use_sympy.tex [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+  similar features for free, which Wolfram\textbar{}Alpha only offers 
+  to its paid users.
+\end{itemize}
+Every input query from the user on SymPy Gamma is first, parsed by its
+own parser, which handles several different forms of function names,
+which SymPy as a library doesn't support. For instance, SymPy Gamma
+supports queries like \texttt{sin\ x}, whereas SymPy doesn't support
+this, and supports only \texttt{sin(x)}.
+
+This parser converts the input query to the resultant SymPy readable
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: resultant --> equivalent

File: other_projects_that_use_sympy.tex [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+\end{itemize}
+Every input query from the user on SymPy Gamma is first, parsed by its
+own parser, which handles several different forms of function names,
+which SymPy as a library doesn't support. For instance, SymPy Gamma
+supports queries like \texttt{sin\ x}, whereas SymPy doesn't support
+this, and supports only \texttt{sin(x)}.
+
+This parser converts the input query to the resultant SymPy readable 
+code, which is then eventually processed by SymPy and the result is 
+finally formatted in LaTeX and displayed on the SymPy Gamma web-app.
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: web-app --> web application

File: other_projects_that_use_sympy.tex [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+
+\subsection{SymPy Live}\label{sympy-live}
+
+SymPy Live is an online web-based Python shell, which runs on Google
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: `web-based` seems redundant

File: other_projects_that_use_sympy.tex [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+App Engine, that executes SymPy code. It is integrated in the SymPy
+documentation examples, located at this \href{http://docs.sympy.org/latest/index.html}{link}.
+
+This is accomplished by providing a HTML/JavaScript GUI for entering
+source code and visualization of output, and a server part which
+evaluates the requested source code. It's an interactive AJAX shell,
+that runs SymPy code using Python on the server.
+
```
Comment (aktech) [inline other_projects_that_use_sympy.tex]: This empty line doesn't leaves a blank line, see [this](https://www.sharelatex.com/learn/Line_breaks_and_blank_spaces#Line_breaks).

File: other_projects_that_use_sympy.tex [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+  It can be run as a standalone app or in an existing app as an
+  admin-only handler, and can also be used for system administration
+  tasks, as an interactive way to try out APIs, or as a debugging aid
+  during development.
+\item
+  It can also be used to plot figures (\href{http://live.sympy.org/?evaluate=from\%20sympy\%20import\%20symbols\%0Afrom\%20sympy.plotting\%20import\%20textplot\%0Ax\%20\%3D\%20symbols(\%27x\%27)\%0Atextplot(x**2\%2C0\%2C5)\%0A\%23--\%0A}{link}), 
+  and execute all kinds of expressions that SymPy can evaluate.
+\end{itemize}
+
+SymPy Live also formats the output in LaTeX for pretty-printing the
```
Comment (aktech) [inline other_projects_that_use_sympy.tex:138]: Add it to \item as well

**Comment by aktech** [comment]
Please fix conflicts.

**Comment by ashutoshsaboo** [comment]
Hi @aktech . I have resolved the conflicts here. Also, I have made the required changes. So, ready for merge now?

I would suggest to merge #31 , before this. :smiley:

**Comment by moorepants** [comment]
#31 was merged, please update this one.

**Comment by ashutoshsaboo** [comment]
@moorepants  So, I have resolved all the conflicts, this passes all the test cases. So, I guess this is ready for merge into the first draft, so that everything can be reviewed soon. Sorry for the delay. 

Thanks. :smile:

**Pull request merged**